app/views/mobile-client-download-url.html

```diff
@@ -0,0 +1,11 @@
+<div ng-if="$ctrl.showPanel" class="mobile-client-download-url col-md-12">
+  <div ng-if="!$ctrl.mobileBuild.metadata.annotations[$ctrl.BUILD_DOWNLOAD_ANNOTATION]" class="spinner spinner-lg" aria-hidden="true"></div>
```

Comment:
There's a filter for getting an annotation that can be used in the html which might shorten this a bit. 
Alternatively, what about extracting this to a variable like `isAnnotated` in the controller, seeing as its being used quite a bit?